src/assets/locales/en/translation.json

```diff
@@ -522,6 +522,7 @@
     "select_account": "Please select an account",
     "install_wallet": "Please install supported wallet",
     "select_wallet": "Please select a wallet",
+    "exclude_us_citizens": "By proceeding you confirm that you have read and accepted the <1>terms and conditions</1>, and represent and warrant that you are not a Resident of the United States or a \"U.S. person\" within the meaning of Rule 902(k) under the United States Securities Act of 1933 (the \"Securities Act\").",
```

Comment:
I assume this is her for completeness, given we're using the Trans component?

Reply:
I did not look into its usage deeply but just quickly checked some similar examples.

Reply:
Sounds good - I like having the dictionary item even if we're not using it (we have a task in the backlog to work out how to use our dictionary files with the Trans component).